Tests/Tests.csproj

```diff
@@ -28,6 +28,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <NoWarn>CS0618</NoWarn>
```

Comment:
The Obsolete takes you placed on the JsonApiMediaTypeFormatter generated compiler warnings in the unit tests you made for them.  This suppresses those warnings in the unit test project and leaves a cleaner build.  It's possible this could be done at the file level, this was just readily available in the VS project settings.